docs/images/FindALostTRNDiagram.png:Zone.Identifier

```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
+LastWriterPackageFamilyName=Microsoft.ScreenSketch_8wekyb3d8bbwe
+ZoneId=3

```

Comment:
What's this file for? Do we need it in version control?

Reply:
Sorry mate, I have no idea
let me remove it - thanks for letting me know